### DISCUSSION TITLE
Configure spacing between messages

### DISCUSSION THREAD

**Discussion by provokateurin** [flyerhq/flutter_chat_ui#654] [Ideas & Requests]
## Is your feature request related to a problem?

For me the spacing between the messages is a little big. I'd like to be able to configure the spacing myself.

## What solution would you suggest?

Introduce option to set the spacing between messages.

**Comment by demchenkoalex** [comment]
Unfortunately impossible to do with the current chat setup. Animations made me split spacings everywhere to different places. If I will have a solution how to animate messages properly and make them work with a spacing set in 1 place I will do it configurable. But this is a long stretch, cause I don't know how to do animations in other way. This is blocking spacings, time outside the bubble and, generally, code quality.

**Comment by provokateurin** [comment]
Oh I see. Thanks anyways :)
